### DISCUSSION TITLE
List of Guns in MW3

### DISCUSSION THREAD

**Discussion by jamesallnutt** [carlssonemil/interstellar#2] [General]
I think we can start prepping the work that needs doing ahead of the official camo list, with the following list:

AR:
- SVA 545 (AR)
- MTZ-556 (AR)
- Holger 556 (AR)
- MCW (AR)
- DG-50 (AR)
- FR 5.56 (AR)

BR:
- BAS-B (BR)
- Sidewinder (BR)
- MTZ-762 (BR)

SMG:
- Striker (SMG)
- WSP Swarm (SMG)
- AMR9 (SMG)
- WSP-9 (SMG)
- Rival-9 (SMG)
- Striker 9 (SMG)

Shotgun:
- Lockwood 680 (Shotgun)
- Haymaker (Shotgun)
- Riveter (Shotgun)

LMG:
- Pulemyot 762 (LMG)
- DG-58 LSW (LMG)
- Holger 26 (LMG)
- Bruen MK9 (LMG)
- TAQ Eradicator (LMG)

Marksman:
- KVD Enforcer (Marksman)
- MCW 6.8 (Marksman)
- DM56 (Marksman)
- MTZ Interceptor (Marksman)

Sniper:
- KATT-AMR (Sniper)
- Longbow (Sniper)
- KV Inhibitor (Sniper)

Handgun:
- COR-45 (Handgun)
- Renetti (Handgun)
- TYR (Handgun)
- WSP Stinger (Handgun)

Launcher:
- RGL-80 (Launcher)

Melee:
- Gutter Knife (Melee)
- Karambit (Melee)

**Comment by carlssonemil** [comment]
Nice! As I mentioned in the [other discussion thread](https://github.com/carlssonemil/mw3/discussions/1) we might not need to create a new tracker site for MW3 and simply just re-design & re-structure the Orion tracker site a bit and add the new guns and camouflages there. I'll prep a branch on there in the coming days for the potential new additions!

Thanks for gathering a list of the new guns! ❤️
